runtime/afl.c

```diff
@@ -87,7 +89,11 @@ CAMLexport value caml_setup_afl(value unit)
 
   shm_id_str = caml_secure_getenv("__AFL_SHM_ID");
   if (shm_id_str == NULL) {
-    /* Not running under afl-fuzz, continue as normal */
+    /* Not running under afl-fuzz.  Allocate space for the bitmap
+       (the instrumented OCaml code will write into it),
+       and continue as normal. */
+    caml_afl_area_ptr = caml_stat_alloc(INITIAL_AFL_AREA_SIZE);
+    memset(caml_afl_area_ptr, 0, INITIAL_AFL_AREA_SIZE);
```

Comment:
The dummy version of `caml_setup_afl` above at ~L35 also needs to do this, or the original bug in #10107 returns on Windows. (`<string.h>` and `"caml/memory.h"` also have to move up above the `#if`)

Reply:
Right!  Thanks for the reminder, I had forgotten about this.  Should be OK now.